tests/console/searchindex/base.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+use phpbb\config\config;
+use phpbb\di\service_collection;
+use phpbb\language\language;
+use phpbb\log\log;
+use phpbb\posting\post_helper;
+use phpbb\search\search_backend_factory;
+use phpbb\search\state_helper;
+use phpbb\user;
+
+require_once __DIR__ . '/mock/search_backend_mock.php';
+
+class base extends phpbb_test_case
```

Comment:
Should be:
```suggestion
class phpbb_console_searchindex_base_test extends phpbb_test_case
```
and file renamed accordingly